installer/mac/pg/src/makefile

```diff
@@ -82,15 +75,16 @@ $(SHAREDPREFIX)/lib/libssl.dylib: openssl-${OPENSSL_VERSION}/Makefile
 openssl-${OPENSSL_VERSION}/Makefile: openssl-${OPENSSL_VERSION}/Configure
 	cd openssl-${OPENSSL_VERSION} && ./Configure --prefix="${SHAREDPREFIX}" darwin64-x86_64-cc zlib no-asm no-krb5 shared
 
-openssl-${OPENSSL_VERSION}/Configure: openssl-${OPENSSL_VERSION}.tar.gz
-	$(TAR) openssl-${OPENSSL_VERSION}.tar.gz
+openssl-${OPENSSL_VERSION}/Configure: openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}
+	$(TAR) openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}
 	touch $@
 
-openssl-${OPENSSL_VERSION}.tar.gz:
+openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}:
 	$(CURL) "https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz"
+	ruby ../../tools/renamesha256.rb openssl-${OPENSSL_VERSION}.tar.gz
```

Comment:
Consider using the environment variable `$CHAIN` to avoid `../..` in paths. It's a bit easier for me to read `$(CHAIN)/installer/mac/tools/renamesha256.rb` because it doesn't require me to remember which file I'm looking at and where it is on disk.